rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/TestBookKeeper.java

```diff
@@ -100,6 +101,43 @@ public void testGetDummyClusterManagerValidInstance() throws Exception
         " Expected : " + DummyClusterManager.class + " Got : " + manager.getClass());
   }
 
+  @Test
+  private void testGetCacheDirSizeinMBs()
+          throws IOException, TException
+  {
+    String testDirectory = CacheConfig.getCacheDirPrefixList(conf) + "0" + CacheConfig.getCacheDataDirSuffix(conf);
+    CacheConfig.setBlockSize(conf, 1 * 1024 * 1024);
+    String backendFileName = testDirectory + "testBackendFile";
+    DataGen.populateFile(backendFileName, 1, 10000000);
```

Comment:
Can we check whether we can work with a smaller file. 100mb file is a big file for unit test